### PR TITLE
Add supported series and unit counts to user profile.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

If you don't have charms/bundles of your own, you may need to edit `user-profile.js` to pass a hardcoded username (like 'lazypower') into `charmstore.list()` instead of the `username` variable.

1. Pull up the user profile.
2. Verify that the unit count (for bundles) and series (for charms) are now displayed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4502/
Test FAILed.

**Comment by hatched** [comment]
:+1: :ok:
Thanks for this update!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4507/
Test PASSed.

File: jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js [hunk 1]
```diff
@@ -347,6 +350,9 @@ var module = module;
         }, this);
       }
       if (processed.entityType === 'bundle') {
+        if (meta['bundle-unit-count']) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js]: I don't think that this is the correct way to protect against this, what will the component display in the event this isn't defined?

Comment (kadams54) [inline jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js]: Nothing is displayed. That causes problems with our layout, but I think that's more a function of CSS than JS (i.e., our table cells need to remain the same width even when they don't have content).

**Comment by kadams54** [comment]
Thanks @hatched! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**